google-harness-engineering.html

```
@@ -0,0 +1,2392 @@
+              <div class="dot" style="background: #fbbc04"></div>
+              <div class="dot" style="background: #34a853"></div>
+            </div>
+            <span>stub_example.py — 固定値を返すだけの替え玉</span>
+          </div>
+          <pre><span class="cmt"># なぜスタブを使うか:
+# 本物の天気APIを呼ぶと「ネットワーク障害」「APIキー切れ」でテストが落ちるため
+# 替え玉を使ってテスト対象コードの判断ロジックのみをテストする</span>
+
+<span class="kw">class</span> <span class="fn">StubWeatherApi</span>:
+    <span class="str">"""本物のWeatherApiの替え玉。常に晴れを返す。"""</span>
+    <span class="kw">def</span> <span class="fn">get_weather</span>(self, city: str) -> str:
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)